README.md

```diff
@@ -361,37 +357,36 @@ application developer were to handle this data, they could rely on the same sche
 rather than their own. Over time, as more and more services use JSON-LD, data representations across
 services would begin to unify to improve cross-service data interoperability.
 
-Right now, each application or service has its own model for representing users. For example, one
-site might use `birthday` as the key for the user's birthday, while others use `day_of_birth` or
-`birthDay`. The sites may even use different formats for the value of the key, with some using
-`YYYY-MM-DD` and another using `DD-MM-YYYY`. Custom logic would have to be written to handle mapping
-the different keys to each other, but also to convert their values to a standard format even though
-they all have the same semantic meaning. They all describe when a person was born.
+Take, for example, our previous user model. Right now, each application or service might use their
+own model definitions: one site could choose `birthday`, while another uses `day_of_birth` or
+`birthDay`, to represent the user's birthday. These models might also be of different formats: some
+could be in `YYYY-MM-DD` while others `DD-MM-YYYY`. Despite all of them containing the same semantic
+meaning, custom logic would have to be written to not only handle the mapping between the different
+keys, but also to convert their values to a standard format.
 
-JSON-LD solves these problems by introducing a `context` to each data model allowing for:
+JSON-LD's `context` solves these problems by allowing for:
 
-1. A unified mapping of keys by complying to base schemata
-1. Value-validation on a [primitive data types level](https://www.w3.org/TR/rdf11-concepts/#section-Datatypes)
+1. A unified mapping of keys that comply to base schemata; and
+1. Value validation for [primitive data types](https://www.w3.org/TR/rdf11-concepts/#section-Datatypes).
 
 
-Going back to the Andy Warhol example, we need to explain how JSON-LD magically maps our
-self-defined keys (`givenName`, `familyName` and `birthDate`) to the properties of schema.org's
-`Person`. If you look at schema.org's `Person` definition, you'll see we didn't choose random key
-names. Names were already part of the definition. In this case, a JSON-LD parser can automatically
-map and execute validation against these properties by using the schema definition.
+To see how `context` achieves this, we need to explain how JSON-LD magically maps our example's
+self-defined keys (`givenName`, `familyName` and `birthDate`) to their matching properties on the
+`Person` schema. If you look at the [`Person` definition](http://schema.org/Person), you'll notice
+that we didn't choose random keys–these keys were already part of the schema definition. Because of
```

Comment:
"–**these keys** were already part of the schema definition"

Replace with "they"